### ISSUE TITLE
3MF support for Canvas3D.io

### ISSUE THREAD

**Issue by GavBak** [MosaicManufacturing/canvas-issues#237]
**Is your feature request related to a problem? Please describe.**

Would like to use 3mf files, canvas throws an error when I upload.

**Describe the solution you'd like**

Update canvas to utilize 3mf

**Describe alternatives you've considered**

**Additional context**

**Comment by Piddey** [comment]
3MF file support would be great! STL is a little bit outdate, isn't it?

**Comment by GavBak** [comment]
3MF is what many people slice to now, without 3MF compatibility there are many files we cannot use in canvas.

**Comment by Piddey** [comment]
I totally agree!
I also publish my stuff in 3mf along with the STL files. The download counts show, that the 3mf files are more popular.
Also 3mf provides some benefits - starting with the impossibility of having non-manifold surfaces. Especially for multi-color and multi-part printing 3mf would be a great enhancement.

**Comment by HauserGer** [comment]
Agreed! Can't even believe it does not support 3MF files. I am returning my Palette 3 Pro due to this, not joking.

**Comment by Bloomingdale-Robotics** [comment]
Need 3MF support

**Comment by Strell69** [comment]
There are Tribe files that use .3mf with colors already added saving lots of time having to color manually in Canvas. Adding .3mf is very useful.

**Comment by moberli** [comment]
Agree - 3mf is coming up more and more..
